fordel-domene/src/main/java/no/nav/foreldrepenger/mottak/felles/LogSettings.java

```diff
@@ -0,0 +1,16 @@
+package no.nav.foreldrepenger.mottak.felles;
```

Comment:
Trenger vi egen klasse for dette ? Kan vel bo i MDCOperations?

Reply:
Burde nok det. Men vil først hindre at det blir flere feil som må håndteres manuelt.